tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```diff
@@ -125,31 +125,33 @@ Services:
       - Azure.Storage.Blobs: source
     Java:
       Project: sdk/storage/azure-storage-perf
-      PrimaryPackage: azure-storage-blob
+      PrimaryPackage: 'com.azure:azure-storage-blob'
       PackageVersions: &java-storage-package-versions
-      - azure-storage-blob: 12.19.0
-        azure-storage-blob-cryptography: 12.18.0
-        azure-storage-file-share: 12.15.0
-        azure-storage-file-datalake: 12.12.0
-        azure-core-http-netty: 1.12.4
-        azure-core-http-okhttp: 1.11.1
-        azure-core: 1.31.0
-        reactor-core: 3.4.21
-      - azure-storage-blob: source
-        azure-storage-blob-cryptography: source
-        azure-storage-file-share: source
-        azure-storage-file-datalake: source
-        azure-core-http-netty: source
-        azure-core-http-okhttp: source
-        azure-core: source
-        reactor-core: source
+      - 'com.azure:azure-storage-blob': 12.19.0
+        'com.azure:azure-storage-blob-cryptography': 12.18.0
+        'com.azure:azure-storage-file-share': 12.15.0
+        'com.azure:azure-storage-file-datalake': 12.12.0
+        'com.azure:azure-storage-common': 12.18.0
+        'com.azure:azure-core': 1.31.0
+        'com.azure:azure-core-http-netty': 1.12.4
+        'com.azure:azure-identity': 1.5.4
```

Comment:
@alzimmermsft: I copied these changes from https://github.com/Azure/azure-sdk-tools/pull/3516.  I think it might be a mistake to include `azure-identity` in the list of storage packages, since I don't believe the storage tests depend on identity.  The package doesn't appear to be used per `mvn dependency:list`:

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1833917&view=logs&j=dc42670d-1473-57cd-faea-36a5bc05eab2&t=ec0ab432-542b-597e-188b-db437ba8e065&l=34